content/en/dynamic_instrumentation/symdb/java.md

```diff
@@ -70,7 +65,26 @@ java \
    $ java -jar my-service.jar -javaagent:dd-java-agent.jar ...
    ```
 
-2. After starting your service with Symbol Database enabled, you can use Symbol Database's IDE-like features on the [APM > Dynamic Instrumentation page][4].
+3. After starting your service with Dynamic Instrumentation with autocomplete and search enabled, you can use Dynamic Instrumentation's IDE-like features on the [APM > Dynamic Instrumentation page][4].
+
+## Additional configuration
+
+### Third party detection
+
+If your code is accidentally recognized as third-party code or third-party code is accidentally recognized as user code, you can configure the third-party detection with these settings:
```

Comment:
I think we should be more concrete here since the reader won't be necessarily understand what we are talking about or why they should even care? Maybe we should go with something like "If autocomplete suggestions do not appear for your package/module, it may have been errantly recognized as third party code..."

Reply:
+1 Maybe change the title to be more clear why the user should care:
"third party detection" -> "autocomplete only my code"

explain to the user we only upload metadata for only code detected as user-code by using hureistc to detect third party code. the library will not upload metadata for third party code...

Reply:
This has been updated by @brett0000FF in [this commit](https://github.com/DataDog/documentation/pull/23635/commits/e4736440f0a189992bfd47ff87fb15875be0a5d7). Let me know if this sounds good to you.